### ISSUE TITLE
Bug: Location object has Country name in lowercase

### ISSUE THREAD

**Issue by SYNchroACK** [MISP/misp-stix#34]
### Expected behavior

Use the field `description` from the MISP event to fill the `name` field in STIX object.

```
{
    "type": "location",
    "spec_version": "2.1",
    "id": "location--84668357-5a8c-4bdd-9f0f-6b50b2555341",
    "created": "2022-09-15T11:29:38.000Z",
    "modified": "2022-09-15T11:29:38.000Z",
    "name": "United States",
    "description": "Country meta information based on the database provided by geonames.org.",
    "country": "['US']",
    "labels": [
        "misp:galaxy-name=\"Country\"",
        "misp:galaxy-type=\"country\"",
        "misp-galaxy:country=\"united states\""
    ],
    "x_misp_Capital": [
        "Washington"
    ],
    "x_misp_Continent": [
        "NA"
    ],
    "x_misp_CurrencyCode": [
        "USD"
    ],
    "x_misp_CurrencyName": [
        "Dollar"
    ],
    "x_misp_ISO": [
        "US"
    ],
    "x_misp_ISO3": [
        "USA"
    ],
    "x_misp_Languages": [
        "en-US,es-US,haw,fr"
    ],
    "x_misp_Population": [
        "310232863"
    ],
    "x_misp_tld": [
        ".us"
    ]
}
```

### Actual behavior

```
{
    "type": "location",
    "spec_version": "2.1",
    "id": "location--84668357-5a8c-4bdd-9f0f-6b50b2555341",
    "created": "2022-09-15T11:29:38.000Z",
    "modified": "2022-09-15T11:29:38.000Z",
    "name": "united states",
    "description": "Country meta information based on the database provided by geonames.org. | United States",
    "country": "['US']",
    "labels": [
        "misp:galaxy-name=\"Country\"",
        "misp:galaxy-type=\"country\"",
        "misp-galaxy:country=\"united states\""
    ],
    "x_misp_Capital": [
        "Washington"
    ],
    "x_misp_Continent": [
        "NA"
    ],
    "x_misp_CurrencyCode": [
        "USD"
    ],
    "x_misp_CurrencyName": [
        "Dollar"
    ],
    "x_misp_ISO": [
        "US"
    ],
    "x_misp_ISO3": [
        "USA"
    ],
    "x_misp_Languages": [
        "en-US,es-US,haw,fr"
    ],
    "x_misp_Population": [
        "310232863"
    ],
    "x_misp_tld": [
        ".us"
    ]
}
```

![Screenshot from 2023-03-04 23-38-01](https://user-images.githubusercontent.com/4184292/222931701-2c1bf7de-df5b-4b0a-8982-a799e74ab0b9.png)

### Steps to reproduce

Process a country associated to an attribute.

### Version

2.4.168

### Python version

3.10.9

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by chrisr3d** [comment]
It should be OK now with https://github.com/MISP/misp-stix/commit/7ed44f0153f0cbac8ed9692c188f3279c7279018, but let me know if there is any further issue with the Location objects, and thank you for reporting this one

**Comment by chrisr3d** [comment]
And better without the typo that is fixed with 08c5a69 ...
